app/forms/waste_carriers_engine/manual_address_form.rb

```diff
@@ -8,15 +8,23 @@ class ManualAddressForm < BaseForm
     # We pass the following attributes in to create a new Address
     attr_accessor :house_number, :address_line_1, :address_line_2, :town_city, :postcode, :country
 
+    validates :house_number, presence: true, length: { maximum: 200 }
+    validates :address_line_1, presence: true, length: { maximum: 160 }
+    validates :address_line_2, length: { maximum: 70 }
+    validates :town_city, presence: true, length: { maximum: 30 }
+    validates :postcode, length: { maximum: 30 }
+    validates :country, presence: true, if: :overseas?
+    validates :country, length: { maximum: 50 }
+
     def initialize(transient_registration)
       super
       # We use this for the correct microcopy and to determine what fields to show
```

Comment:
NIT Adding a line between super and this as per other changes

```suggestion
      
      # We use this for the correct microcopy and to determine what fields to show
```